parachains/runtimes/assets/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -387,6 +392,15 @@ pub type AssetFeeAsExistentialDepositMultiplierFeeCharger = AssetFeeAsExistentia
 	TrustBackedAssetsInstance,
 >;
 
+/// Cases where a remote origin is accepted as trusted Teleporter:
+/// - teleportation of DOT from the parent Relay Chain and sibling system parachains; and
+/// - teleportation of sibling parachain's assets (as ForeignCreators).
+pub type TrustedTeleporters = (
+	NativeAsset,
```

Comment:
Same question here -- remove `NativeAsset` and keep `ConcreteNativeAssetFromSiblingSystemParachain`?